sanicap/sanicap.py

```diff
@@ -217,7 +219,7 @@ def get_ip(self, address):
             return address
 
 
-def sanitize(filepath_in, filepath_out = None, sequential=True, ipv4_mask=0, ipv6_mask=0, mac_mask=0, start_ipv4='10.0.0.1', start_ipv6='2001:aa::1', start_mac='00:aa:00:00:00:01', info=True):
+def sanitize(filepath_in, filepath_out = None, sequential=True, append=False, ipv4_mask=0, ipv6_mask=0, mac_mask=0, start_ipv4='10.0.0.1', start_ipv6='2001:aa::1', start_mac='00:aa:00:00:00:01', fixed_vlan=None, info=True):
```

Comment:
In lieu of an `Options` or `Config` class for all of these options, making them kwarg only should help usage stay readable.

```suggestion
def sanitize(filepath_in, *, filepath_out = None, sequential=True, append=False, ipv4_mask=0, ipv6_mask=0, mac_mask=0, start_ipv4='10.0.0.1', start_ipv6='2001:aa::1', start_mac='00:aa:00:00:00:01', fixed_vlan=None, info=True):
```